docs/src/library/public.md

```diff
@@ -11,6 +11,13 @@ Modules = [ClimaSeaIce]
 Private = false
 ```
 
+## ClimaSeaIce.SeaIceThermodynamics
+
+```@autodocs
+Modules = [ClimaSeaIce.SeaIceThermodynamics]
+Private = false
+```
+
```

Comment:
@simone-silvestri I think this will heal the error you get...

there is another way to add a kwarg in `makedocs` -> `warnonly = [:cross_references]` so that documenter doesn't check that all the docstrings are in the docs

Reply:
Thanks. I think we are having the same problem with ClimaOcean. I ll try to correct it